tests/components/recorder/test_history.py

```diff
@@ -25,9 +25,11 @@
 from homeassistant.helpers.json import JSONEncoder
 import homeassistant.util.dt as dt_util
 
-from .common import async_wait_recording_done, wait_recording_done
-
 from tests.common import SetupRecorderInstanceT, mock_state_change_event
+from tests.components.recorder.common import (
```

Comment:
This seems off? This should be relative, right?

Reply:
I have moved the recorder changes out to a separate PR, as there is a problem with recorder tests when they are run standalone: see https://github.com/home-assistant/core/pull/78734